enums.rb

```diff
@@ -0,0 +1,133 @@
+# rubocop:disable Metrics/ModuleLength
+module Enumerables
+  def my_each
+    if block_given?
+      each do |element|
+        yield(element)
+      end
```

Comment:
You're not allowed to use #each method

Reply:
@Dandush03 hello this each was automated by rubocop what I did was set a for on this and yield the value 😕 
```
for item in self
yield (value)
end
```
when I use rubocop it gets changed hope you can give me help on that :)

Reply:
Followed to solve it with `# rubocop:disable Style/For`